src/applications/edu-benefits/10203/tests/edu-10203.cypress.spec.js

```diff
@@ -0,0 +1,73 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['minimal-test', 'confirmation-stem-test'],
+    fixtures: { data: path.join(__dirname, 'schema') },
+
+    pageHooks: {
+      introduction: ({ afterHook }) => {
+        afterHook(() => {
+          cy.findAllByText(/start.+without signing in/i, { selector: 'button' })
+            .first()
+            .click();
+        });
+      },
+
+      'benefits/initial-confirm-eligibility': () => {
+        cy.findByText(/Continue/i, { selector: 'button' }).click();
+      },
+
+      'benefits/stem-eligibility': () => {
+        cy.get('@testKey').then(testKey => {
+          if (testKey === 'confirmation-stem-test') {
+            cy.get('input[id="root_isEnrolledStemNo"]').click();
+            // eslint-disable-next-line cypress/no-unnecessary-waiting
```

Comment:
Is this waiting for a request? If the next step is delayed because it's waiting on a response, you can [wait for that response if you've stubbed it](https://docs.cypress.io/api/commands/wait.html#Alias). That's preferable to waiting by a static length of time.

If your test would otherwise fail at this point without the wait, and you see a `XHR` (instead of `XHR Stubbed`) right before this point, that's a good indicator that it's timed out on a request that wasn't stubbed. And in that case, usually the fix is to stub out the response to that request so it can return immediately and update the UI.

Reply:
The waiting is due to a slide down reveal animation for the `is pursuing teaching cert` question that needed to run before clicking on an answer. I did find that I can set the  `waitForAnimations` configuration in the spec file but wasn't sure if that would be an acceptable solution.

Reply:
@U-DON I pushed in the change for you to compare.

Reply:
I think that's acceptable. The default value of `false` is just to speed up tests, namely the very long form tests that don't depend on the animation. Thanks!